recruitment/views.py

```diff
@@ -205,7 +210,8 @@ def json(self):
                 'monocolor': self.monocolor,
                 'charts': self.charts,
                 'sorted_values': self.sorted_values(),
-                'sorted_value_counts': self.sorted_value_counts()
+                'sorted_value_counts': self.sorted_value_counts(),
+                'y_limit': self.y_limit
```

Comment:
Add a comma at the end of this line in case someone wants to add an item in the future.